apps/web/app/(auth)/account/keys/create-api-key-form.tsx

```diff
@@ -0,0 +1,78 @@
+"use client";
+
+import { SubmitButton } from "@/features/form/loading-button";
+import { Button } from "@workspace/ui/components/button";
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+} from "@workspace/ui/components/card";
+import { Input } from "@workspace/ui/components/input";
+import { Label } from "@workspace/ui/components/label";
+import { useState } from "react";
+
+interface CreateApiKeyFormProps {
+  onSubmit: (formData: FormData) => Promise<void>;
+}
+
+export function CreateApiKeyForm({ onSubmit }: CreateApiKeyFormProps) {
```

Comment:
don't need this, use dialog-manager with a input to get the value and completly remove this